tests/components/threshold/test_binary_sensor.py

```diff
@@ -29,54 +46,39 @@ async def test_sensor_upper(hass: HomeAssistant) -> None:
     assert await async_setup_component(hass, "binary_sensor", config)
     await hass.async_block_till_done()
 
-    # Set the monitored sensor's state to the threshold
-    hass.states.async_set("sensor.test_monitored", 15)
-    await hass.async_block_till_done()
-    state = hass.states.get("binary_sensor.threshold")
-    assert state.attributes["position"] == "below"
-    assert state.state == "off"
-
-    hass.states.async_set(
-        "sensor.test_monitored",
-        16,
-        {ATTR_UNIT_OF_MEASUREMENT: UnitOfTemperature.CELSIUS},
-    )
+    hass.states.async_set("sensor.test_monitored", from_val)
     await hass.async_block_till_done()
     state = hass.states.get("binary_sensor.threshold")
     assert state.attributes["entity_id"] == "sensor.test_monitored"
-    assert state.attributes["sensor_value"] == 16
-    assert state.attributes["position"] == "above"
     assert state.attributes["upper"] == float(config["binary_sensor"]["upper"])
     assert state.attributes["hysteresis"] == 0.0
     assert state.attributes["type"] == "upper"
-    assert state.state == "on"
-
-    hass.states.async_set("sensor.test_monitored", 14)
-    await hass.async_block_till_done()
-    state = hass.states.get("binary_sensor.threshold")
-    assert state.attributes["position"] == "below"
-    assert state.state == "off"
 
-    hass.states.async_set("sensor.test_monitored", 15)
+    hass.states.async_set("sensor.test_monitored", to_val)
     await hass.async_block_till_done()
     state = hass.states.get("binary_sensor.threshold")
-    assert state.attributes["position"] == "below"
-    assert state.state == "off"
+    assert state.attributes["position"] == expected_position
```

Comment:
I'm not sure but probably there are also constants for accessing state attributes, that would be nice

Reply:
Yes. I've updated to use the constants from const.py. There are more to more from binary_sensor.py -> const.py, but I would prefer to do that in another PR.